lib/utils/getEstimatedDailyRebases/index.ts

```diff
@@ -0,0 +1,9 @@
+import { EPOCH_INTERVAL } from "../../constants/index";
+
+// Calculates estimated daily rebases based on current block rate
+export async function getEstimatedDailyRebases(blockRate: number) {
```

Comment:
this has no `await` so it doesn't need to be an `async` function. You can remove it out of the promises array here 
https://github.com/KlimaDAO/klimadao/pull/70/files#diff-b96f585d57b9419336b96f03d5d29b05d4b7d5855d5162ab0d73d2cca3d8afbdR59

And remove the `awaits` where used